tests/test_downscoped.py

```diff
@@ -383,3 +410,256 @@ def test_to_json(self):
                 ]
             }
         }
+
+
+class TestCredentials(object):
+    QUOTA_PROJECT_ID = "QUOTA_PROJECT_ID"
+    GRANT_TYPE = "urn:ietf:params:oauth:grant-type:token-exchange"
+    REQUESTED_TOKEN_TYPE = "urn:ietf:params:oauth:token-type:access_token"
+    TOKEN_EXCHANGE_ENDPOINT = "https://sts.googleapis.com/v1/token"
+    SUBJECT_TOKEN_TYPE = "urn:ietf:params:oauth:token-type:access_token"
+    SUCCESS_RESPONSE = {
+        "access_token": "ACCESS_TOKEN",
+        "issued_token_type": "urn:ietf:params:oauth:token-type:access_token",
+        "token_type": "Bearer",
+        "expires_in": 3600,
+    }
+    ERROR_RESPONSE = {
+        "error": "invalid_grant",
+        "error_description": "Subject token is invalid.",
+        "error_uri": "https://tools.ietf.org/html/rfc6749",
+    }
+    CREDENTIAL_ACCESS_BOUNDARY_JSON = {
+        "accessBoundary": {
+            "accessBoundaryRules": [
+                {
+                    "availablePermissions": AVAILABLE_PERMISSIONS,
+                    "availableResource": AVAILABLE_RESOURCE,
+                    "availabilityCondition": {
+                        "expression": EXPRESSION,
+                        "title": TITLE,
+                        "description": DESCRIPTION,
+                    },
+                }
+            ]
+        }
+    }
+
+    @classmethod
```

Comment:
`@classmethod` is an unusual choice here:  it implies that you expect to override the current class, supplying different class-level attribute values (or methods).  Given that you don't use `cls` here, I think you probably want `@staticmethod` instead.

Reply:
Done.